Assignment.txt

```diff
@@ -0,0 +1,15 @@
+Armstrong Park This is my theory homework
+1.
+a.O(n)
+b.2(n)
```

Comment:
Not sure what you meant by this, but either way, the time complexity of this is n^2 + n, which evaluates to O(n^2). The n^2 comes from the nested for loops in the second half of the code snippet and the n comes from the first for loop.